### DISCUSSION TITLE
Inviting External Members to University of Illinois GitHub Organization

### DISCUSSION THREAD

**Discussion by zzrcxb** [uillinois-community/uillinois-community.github.io#71] [Q&A] [answered]
I'm attempting to add external members to a GitHub organization for the University of Illinois, but it appears they're required to sign in via the U of I SAML Identity Provider, which they don't have access to. How can I invite external members to the organization without using SAML?

Thanks!

**Comment by jtgorman** [comment]
Hmm, I've not tried to do this at the organization level, but at least in the past when I've added external/outside collaborators to a repository they've been able to access it without the campus 2FA. It's been a while since I've done it.

**Reply by zzrcxb** [reply]
Thanks for your reply, Jon! We are exploring the possibility of using the university GitHub organization for a *cross-institution* research center. Do you think it's possible to disable the SAML requirement for our organization?

**Reply by jtgorman** [reply, marked as answer]
At least that used to be up to the org maintainers, but strongly encouraged, in the settings for the org. Looking at some of the docs though looks like it might be required now?  You might want to reach out to your org maintainers/admins. If they are ok with disabling SAMLit, they probably should reach out to the fire off an email to githubsupport@uillinois.edu for clarification.
